apps/api/src/teams/event-registrations/event-registrations.service.ts

```diff
@@ -71,28 +104,45 @@ export class TeamEventRegistrationsService {
     return await this.teamEventRegistrationRepository.findWithPagination(
       options,
       filter,
-      { orderBy: { createdAt: 'ASC' }, populate: ['user', 'event', 'event.team'] },
+      {
+        orderBy: { createdAt: 'ASC' },
+        populate: ['user', 'event', 'event.team'],
+      },
     );
   }
 
-  public async findOne(user: User, teamEventRegistrationId: number): Promise<TeamEventRegistration> {
-    const registration = await this.teamEventRegistrationRepository.findOneOrFail({ teamEventRegistrationId }, { populate: ['user', 'event', 'event.team'] });
+  public async findOne(
+    user: User,
+    teamEventRegistrationId: number,
+  ): Promise<TeamEventRegistration> {
+    const registration = await this.teamEventRegistrationRepository.findOneOrFail(
+        { teamEventRegistrationId },
+        { populate: ['user', 'event', 'event.team'] },
+      );
 
     if (registration.event.private) {
       const membership = await this.teamMemberRepository.findOne({
         user,
         team: { teamId: registration.event.team.teamId },
       });
       if (!membership)
-        throw new ForbiddenException('Not a team member');
+throw new ForbiddenException('Not a team member');
     }
 
     return registration;
   }
 
-  public async remove(user: User, teamEventRegistrationId: number): Promise<void> {
-    const registration = await this.teamEventRegistrationRepository.findOneOrFail({ teamEventRegistrationId });
-    const event = await this.teamEventRepository.findOneOrFail({ teamEventId: registration.event.teamEventId }, { populate: ['team', 'team.members'] });
+  public async remove(
+    user: User,
+    teamEventRegistrationId: number,
+  ): Promise<void> {
+    const registration = await this.teamEventRegistrationRepository.findOneOrFail({
+        teamEventRegistrationId,
+      });
+    const event = await this.teamEventRepository.findOneOrFail(
+      { teamEventId: registration.event.teamEventId },
+      { populate: ['team', 'team.members'] },
+    );
```

Comment:
Pourquoi l'avoir changé
```suggestion
  public async remove(user: User, teamEventRegistrationId: number): Promise<void> {
    const registration = await this.teamEventRegistrationRepository.findOneOrFail({ teamEventRegistrationId });
    const event = await this.teamEventRepository.findOneOrFail({ teamEventId: registration.event.teamEventId }, { populate: ['team', 'team.members'] });
```